src/main/kotlin/com/terraformation/backend/accelerator/db/ApplicationStore.kt

```diff
@@ -554,7 +554,7 @@ class ApplicationStore(
   }
 
   /** Updates the country code of an application. */
-  private fun updateCountryCode(applicationId: ApplicationId, countryCode: String) {
+  fun updateCountryCode(applicationId: ApplicationId, countryCode: String) {
```

Comment:
Add a permission check here since this could be called from additional places in the future.